File: src/Web/Desnz.Chmm.Web/ClientApp/src/app/stores/onboarding/dtos/view-organisation-dto.ts
export interface ViewOrganisationDto {
  id: string;
  name: string;
  status: string;
  licenceHolders: ViewOrganisationLicenceHolderDto[] | null | undefined;
  isNonSchemeParticipant: boolean;
}

export interface ViewOrganisationLicenceHolderDto {
  name: string;
}

File: src/Api/Identity/Desnz.Chmm.Identity.Common/Dtos/Organisation/ViewOrganisationDto.cs
namespace Desnz.Chmm.Identity.Common.Dtos.Organisation
{
    public class ViewOrganisationDto
    {
        public bool IsNonSchemeParticipant { get; set; }
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Status { get; set; }
        public List<ViewOrganisationLicenceHolderDto> LicenceHolders { get; set; }
    }

    public class ViewOrganisationLicenceHolderDto
    {
        public string Name { get; set; }
    }
}
